### DISCUSSION TITLE
How to open sqlitestudio more one on windows

### DISCUSSION THREAD

**Discussion by suatsuphi** [pawelsalawa/sqlitestudio#4042] [Q&A]
Hi,
I wanna open it more one on windows, two times or more.

**Comment by pawelsalawa** [comment]
If you're using version 3.3.x, then you can enable this in configuration: 
![image](https://user-images.githubusercontent.com/481846/112281403-4e2cd780-8c86-11eb-8c28-b9a46a2ef4e5.png)
